transform.js

```diff
@@ -5,7 +5,17 @@ function toJSON(sdp) {
     const sessionPart = mediaSections.shift();
     const description = {
         media: [],
+        groups: [],
     };
+    SDPUtils.matchPrefix(sessionPart, 'a=group:').forEach((groupLine) => {
+        const parts = groupLine.split(' ');
+        const semantics = parts.shift().substr(8);
+        description.groups.push({
+            semantics,
+            contents: parts,
```

Comment:
not sure if that name is too jingle-y... @legastero?

Reply:
The more accurate one would be `media`?

Reply:
its actually a list of mids so I went for 'mids'